src/Commands/ArtifactCommand.php

```
@@ -318,6 +340,65 @@ protected function doExecute(): void {
+      $this->logger->warning('Unable to list remote branches for cleanup: ' . $exception->getMessage());
+
+      return;
+    }
+
+    $protected_branches = [$this->destinationBranch];
+    $default_branch = $this->repo->getRemoteDefaultBranch($this->remoteName);
+    if ($default_branch !== NULL) {
+      $protected_branches[] = $default_branch;
+    }
+
+    $stale = ArtifactGitRepository::filterStaleBranches($branches, $this->cleanupPattern, $this->cleanupAge * 86400, $this->now, $protected_branches);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\DrevOps\GitArtifact\Git\ArtifactGitRepository' in method 'cleanupStaleBranches'. (undefined)

(StaticAccess)